src/codeu/chat/DerbyStore.java

```diff
@@ -34,90 +57,155 @@ public DerbyStore() {
+			
+			stmt.execute("CREATE TABLE " + chatParticipantsTableName + "(conversationid varchar(255), userid varchar(255))");
 			
-			// Give confirmation of connection
-			System.out.println("Tables exist. Connection made.");
+			// Give confirmation of execution.
+			LOG.info("Tables do not exist. Table creation executed.");
 		}
 		catch (Exception ex) {
-						try {
-						// Create the chat user table
-						stmt.execute("CREATE TABLE chatuser(id varchar(255), name varchar(255), password varchar(255), creation BIGINT)");
-						
-						// Create the conversations table
-						stmt.execute("CREATE TABLE conversation(id varchar(255), "
-								+ "owner varchar(255), creation BIGINT, title varchar(255), users varchar(255), firstMessage varchar(255)," +
-						"lastMessage varchar(255))");
-						
-						// Create the message table
-						stmt.execute("CREATE TABLE message(id varchar(255),"
-								 + "previous varchar(255), creation BIGINT, author varchar(255), content varchar(255), nextMessage varchar(255))");
-						
-						// Give confirmation of execution.
-						System.out.println("Tables do not exist. Table creation executed.");
-						} catch (Exception e) {
-							e.printStackTrace();
-						}
 			ex.printStackTrace();
 		}
 	}
 	
-	public boolean checkForUsername(String username) throws SQLException {
-		return stmt.execute("SELECT * FROM chatuser WHERE password = '" + username + "'" );
+	public boolean checkUsernameExists(String name) throws SQLException {
+		PreparedStatement checkUserTest = conn.prepareStatement(checkUsername);
+		checkUserTest.setString(1, name);
+		ResultSet user = checkUserTest.executeQuery();
+		
+		return user == null;
 	}
 	
-	public boolean checkForPassword(String password) throws SQLException {
-		return stmt.execute("SELECT * FROM chatuser WHERE password = '" + password + "'" );
+	public String userLoginCheck(String name, String password) throws SQLException {
+		PreparedStatement checkValidUserTest = conn.prepareStatement(checkValidUser);
+		checkValidUserTest.setString(1, name);
+		checkValidUserTest.setString(2, password);
+		ResultSet user = checkValidUserTest.executeQuery();
+		
+		// TEST
+		return (user != null) ? user.getString(1) : null;
 	}
 	
 	public void addUser(User u) throws SQLException {
-		stmt.execute("INSERT INTO chatuser VALUES(" + "\'" + removeCharsInUuid(u.id.toString()) + "\'" +
-				"," + "\'" + u.name + "\'" + ", 'test', " + u.creation.inMs() + ")");
+		PreparedStatement addUser = conn.prepareStatement(addUserInfo);
+		addUser.setString(1, removeCharsInUuid(u.id.toString()));
+		addUser.setString(2, u.name);
+		addUser.setString(3, "test");
+		addUser.setLong(4, u.creation.inMs());
+		
+		addUser.executeUpdate();
+		
+		/*stmt.execute("INSERT INTO " + userTableName + " VALUES(" + "\'" + removeCharsInUuid(u.id.toString()) + "\'" +
+				"," + "\'" + u.name + "\'" + ", 'test', " + u.creation.inMs() + ")");*/
```

Comment:
I'm sure you're aware of the commented out stuff here, but I just wanted to point it out so you remember to remove it later.